src/region/RegionAnnotations.tsx

```diff
@@ -65,10 +68,31 @@ export default class RegionAnnotations extends React.PureComponent<Props, State>
 
     handleStop = (shape: Rect): void => {
         const { location, setStaged, setStatus } = this.props;
+        const { drawStarted } = this.state;
+        if (!drawStarted) {
+            return;
+        }
+
         setStaged({ location, shape });
         setStatus(CreatorStatus.staged);
     };
 
+    handleUpdate = (shape: Rect | null): void => {
```

Comment:
I can see the benefit of this because then it would avoid the preview controls button flickering when you click, right? Could we instead adjust the `RegionCreator` `onStart` callback to only be called when the `updateDraw` exceeds the min threshold so that `RegionAnnotations` doesn't need to change and add extra state?